frontend/public/404.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Page Not Found</title>
+
+    <style media="screen">
```

Comment:
Out of curiosity, is there a reason for placing the style information here? 😅

Reply:
These files are auto generated